apps/electron/src/main.js

```diff
@@ -109,8 +109,13 @@ function askToQuit() {
  * Allows processes to escalate errors to be shown in electron
  * @param {string} error
  */
-function escalateError(error) {
-  dialog.showErrorBox('An unrecoverable error occurred', error);
+function escalateError(error, unrecoverable = true) {
```

Comment:
question: I am not sure we can have the unrecoverable defaulting to true now that we shutdown the app